test/collection_test.js

```diff
@@ -402,7 +402,13 @@ describe("Collection", () => {
       beforeEach(() => {
         sandbox.stub(client, "execute").returns(Promise.resolve({
           json: {data: [{a: 1}]},
-          headers: {get: () => {}}
+          headers: {
+            get: (name) => {
+              if (name === "ETag") {
+                return "42";
```

Comment:
The server returns timestamp between quotes `"42"`. This is fine, you should use them as they are. Value between quotes is also accepted on the querystring `?_since="42"`